ldap2date.d.ts

```diff
@@ -0,0 +1,10 @@
+export function parse(time: string): Date;
```

Comment:
`.parse` returns `null` instead of a `Date`. if it fails at parsing the string. Could we change the return type here to be nullable?

Reply:
I'll probably replace that null with a NaN in the future to reduce the number of cases that one needs to check for when using that function. Thanks for updating that.